framer/TextLayer.coffee

```diff
@@ -0,0 +1,123 @@
+{Layer} = require "./Layer"
+{Events} = require "./Events"
+
+class exports.TextLayer extends Layer
+
+	constructor: (options={}) ->
+
+		options.width ?= 300
+		options.backgroundColor ?= "transparent"
+		options.html ?= "Add text"
+		options.color ?= "#888"
+
+		super options
+
+		# Set type defaults
+		if not @fontFamily
+
+			currentDevice = Framer.Device.deviceType
+
+			# Apple Device: SF UI
+			if currentDevice.indexOf("apple") > -1
+				@_setStyle("-apple-system, SF UI Text, Helvetica Neue", @fontSize, @fontWeight, @lineHeight)
+
+			# Google Device: Roboto
+			if currentDevice.indexOf("google") > -1
+				@_setStyle("Roboto, Helvetica Neue", @fontSize, @fontWeight, @lineHeight)
+
+			if currentDevice.indexOf("microsoft") > -1
+				@_setStyle("Segoe UI, Helvetica Neue", @fontSize, @fontWeight, @lineHeight)
+
+			# Else: macOS
+			else
+				@_setStyle("-apple-system, SF UI Text, Helvetica Neue", @fontSize, @fontWeight, @lineHeight)
```

Comment:
There is one, but it doesn't behave how I'd like it to, currently. See:

> Currently not using the recently added Utils.deviceFont utility, because I’d like it to work on a device-level, rather than an OS-level. (So, when working on a Mac, but previewing on a Microsoft device → Segoe UI.)